namespaceFinder.php

```diff
@@ -38,6 +53,10 @@ private function isRelativePath($path){
         return $path[0] == '.';
     }
 
+    private function isPageRelativePath($path){
+        return $path[0] == '~';
```

Comment:
So what happens if $path is empty (or null) here? I'm not sure that you can rule out that this might happen.

Instead of indexing into the string (`$page[0]`) I'd use `substr($page, 0, 1)` which handles these cases gracefully.

Reply:
It's cool that you've been thinking about this corner case; Fortunately it cannot happen, because before calling this method (from `computeWantedNs`) we do this:

```
if($wantedNS == '') {
    $wantedNS = $this->getCurrentNamespace();
}
```
(which sets it to `"."`)
(the case where it would be `null` also cannot happen because of the way this class is instantiated from `syntax.php` in the first place).

Reply:
Ok, fine. If `$path` is guaranteed to be non-empty then my nit doesn't apply.